tests/run-make/extern-flag-disambiguates/rmake.rs

```diff
@@ -0,0 +1,30 @@
+//@ ignore-cross-compile
+
+use run_make_support::{cwd, run, run_in_tmpdir, rustc};
+
+// Attempt to build this dependency tree:
+//
+//	A.1   A.2
+//	 |\    |
+//	 | \   |
+//        B  \  C
+//         \ | /
+//          \|/
+//           D
+//
+// Note that A.1 and A.2 are crates with the same name.
+
```

Comment:
Suggestion: it's useful to have the link to see where the test came from. If it fails, the contributor who caused it to fail can then read on why it exists.
https://github.com/rust-lang/rust/commit/cc3c8bbfaf5af19caf3deb131a995a65ca4674f9